src/Adapter/Profile/Permission/QueryHandler/GetPermissionsForConfigurationHandler.php

```diff
@@ -158,19 +159,29 @@ private function getTabsForPermissionsConfiguration()
     /**
      * @param array $tabs
      * @param int $parentId
+     * @param int $nestingLevel
      *
      * @return array
      */
-    private function buildTabsTree(array &$tabs, $parentId = 0)
+    private function buildTabsTree(array &$tabs, $parentId = 0, $nestingLevel = 1)
```

Comment:
You can init with `0`, and you should use a Class const rather than a variable to store the max nesting level

Reply:
I'm afraid that class is public and can be misused. :/ But if you think it's better, I'll move it to class const.

Reply:
don't forget to use `0` as the initial value